arcsi/view/show.py

```diff
@@ -35,8 +35,24 @@ def view_show(id):
     show = archon_view_show(id)
     if hasattr(show, "status_code") and show.status_code == 404:
         return "Show not found"
-    empty_playlist = show_playlist_is_empty(show.playlist_name)
-    return render_template("show/view.html", show=show, empty_playlist=empty_playlist)
+    az = AzuraArchive(
+        None,
+        None,
+        None,
+        None,
+        None,
+        show.get("playlist_name"),
+    )
+    existing_playlist = az.find_playlist_id()
```

Comment:
This part is hard to understand in terms of what happens here. If empty_playlist is boolean then remove it and use a boolean function on the existing playlist. That's easier to change or extend later.

Reply:
I moved this part into the utils layer and also renamed the functions.

Reply:
Thank you!